flask_config.py

```diff
@@ -1,8 +1,14 @@
 """Flask configuration class."""
 import os
 
+
 class Config:
     """Base configuration variables."""
-    SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
.env.template entry for this can be removed (you also probably want to add new entries for the TRELLO_ ones as well)